src/presentations/part-1/assets/examples/go/nix/go.nix

```diff
@@ -0,0 +1,25 @@
+# We rely on a devenv  module to setup golang.
+# This keeps the config simple, while setting up all the required tooling for vscode etc.
+{pkgs}: [
+  {
+    packages = with pkgs; [
+      curl
+      eza
+      fd
```

Comment:
question: do you have any usage of these tools? maybe just add `fd`, `curl` or `coreutils` and `findutils` which is generally a good idea to do, and maybe `git`, `just`?

Maybe just add `build`, `lint` into a `justfile` (really simple) ?